File: OOP-ClassesAndObjects_HristoChipev/Family.cs
using System;
namespace OOP_Objects_Classes
{
    public class Family
    {
        private List<Person> family;

        public Family(List<Person> people)
        {
            this.family = people;
        }

        public void AddMember(Person person)
        {
            family.Add(person);
        }

        public Person GetOldestMember()
        {
            var oldestPerson = family.First();

            foreach (var person in family)
            {
                if (oldestPerson.Age < person.Age)
                {
                    oldestPerson = person;
                }
            }

            return oldestPerson;
        }

        public void PrintFamily()
        {
            if (family.Count < 1)
            {
                Console.WriteLine("[]");
                return;
            }

            foreach (var person in family)
            {
                if (person != family.Last())
                {
                    Console.Write($"[{person.Name}, {person.Age}], ");
                }
                else
                {
                    Console.Write($"[{person.Name}, {person.Age}]");
                }
            }

            Console.WriteLine();
        }
    }
}

File: OOP_Classes&Objects_Hristiyana_Shopova/src/Family/Family.java
package Family;

import java.util.LinkedList;
import java.util.List;

public class Family {
    List<Person> people = new LinkedList<>();

    public void addMember(Person member)
    {
        people.add(member);
    }

    public Person getOldestMember()
    {
        Person oldest = people.get(0);
        for (int i = 0; i < people.size(); i++) {
            Person currMember = people.get(i);

            if(currMember.getAge() > oldest.getAge())
            {
                oldest = currMember;
            }
        }
        return oldest;
    }

}